src/components/updateCar.ts

```diff
@@ -0,0 +1,42 @@
+import GarageAPI from './GarageAPI';
+
+async function updateCar() {
```

Comment:
I don't think it has correct name, updateCar sounds like an instant action but this code just setups an event handler